### PR TITLE
Teachers Page - New UI

### PR DESCRIPTION
**Pull request by aayush-05**
**Screenshots**
**1.**
image
-----------------------------------------------------------------------------------------------------------------
**2.**
image

_Images have been designed to maintain aspect ratio. The last two images will be replaced with new images with proper dimensions once those pages get done in New UI._

### CODE REVIEW

**Comment by aayush-05** [comment]
@tachyons @nitin10s please have a look.

File: app/assets/stylesheets/logixRelease.scss [hunk 1]
```diff
@@ -235,3 +227,35 @@ a.btn.btn-outline-danger {
...
+  padding-bottom: 40px;
+}
+
+.container-fluid_teachersbackground{
+  background-color: $gray-100;
+  padding-top: 40px;
+  padding-bottom: 40px;
+}
+.image_teachers{
+  max-height: 100%;
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/assets/stylesheets/logixRelease.scss]: Fixed

**Comment by aayush-05** [comment]
Solved merge conflicts, ready to be merged @tachyons

**Comment by tachyons** [comment]
Conflicts

**Comment by aayush-05** [comment]
Done @tachyons

**Comment by aayush-05** [comment]
@tachyons please get it merged before anyone else pushes there UI code, so that conflicts don't arise agan.

**Pull request merged**